db/mysql/data.sql

```diff
@@ -6,12 +6,161 @@ delete from `sch_usr_notification`;
+INSERT INTO `user_role_permission` values((select id from `role` where name='Spravca transformacii'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `permission` VALUES (NULL, 'role.edit', false);
+INSERT INTO `permission` VALUES (NULL, 'role.read', false);
+INSERT INTO `user_role_permission` values((select id from `role` where name='Administrator'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='User'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Povinna osoba'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Spravca transformacii'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `permission` VALUES (NULL, 'role.delete', false);
+INSERT INTO `user_role_permission` values((select id from `role` where name='Administrator'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `permission` VALUES (NULL, 'pipeline.create', false);
+INSERT INTO `user_role_permission` values((select id from `role` where name='Administrator'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='User'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Povinna osoba'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Spravca transformacii'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `permission` VALUES (NULL, 'pipeline.copy', false);
+INSERT INTO `user_role_permission` values((select id from `role` where name='Administrator'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='User'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Povinna osoba'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `user_role_permission` values((select id from `role` where name='Spravca transformacii'), (SELECT max(id) FROM  `permission`));
+INSERT INTO `usr_user` VALUES (NULL,'admin',1,'100000:3069f2086098a66ec0a859ec7872b09af7866bc7ecafe2bed3ec394454056db2:b5ab4961ae8ad7775b3b568145060fabb76d7bca41c7b535887201f79ee9788a','John Admin',20);
+INSERT INTO `usr_extuser` VALUES ((SELECT max(id) FROM  `usr_user`), 'admin');
+INSERT INTO `usr_user` VALUES (NULL,'user',2,'100000:3069f2086098a66ec0a859ec7872b09af7866bc7ecafe2bed3ec394454056db2:b5ab4961ae8ad7775b3b568145060fabb76d7bca41c7b535887201f79ee9788a','John User',20);
+INSERT INTO `usr_extuser` VALUES ((SELECT max(id) FROM  `usr_user`), 'user');
+INSERT INTO `sch_usr_notification` VALUES (NULL,1,1,1),(NULL,2,1,1);
 INSERT INTO `sch_usr_notification_email` VALUES (1,1),(2,2);
+INSERT INTO `usr_user_role` VALUES (1,1),(1,2),(2,1);
```

Comment:
yes you are right, i will correct it.

On Fri, Mar 20, 2015 at 7:36 PM, Tomas Knap <notifications@github.com>
wrote:

> In db/mysql/data.sql
> <https://github.com/UnifiedViews/Core/pull/282#discussion_r26864895>:
>
> >  INSERT INTO `sch_usr_notification_email` VALUES (1,1),(2,2);
> > +INSERT INTO `usr_user_role` VALUES (1,1),(1,2),(2,1);
>
> Is this correct? It seems that user (2) is mapped to admin role (1), which
> is wrong!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UnifiedViews/Core/pull/282/files#r26864895>.
>